omod/src/main/java/org/openmrs/module/chartsearch/web/resources/ChartSearchNoteResource.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See
+ * the License for the specific language governing rights and
+ * limitations under the License.
+ *
+ * Copyright (C) OpenHMIS.  All Rights Reserved.
+ */
+
+package org.openmrs.module.chartsearch.web.resources;
+
+import org.openmrs.api.context.Context;
+import org.openmrs.module.chartsearch.api.ChartSearchService;
+import org.openmrs.module.chartsearch.cache.ChartSearchBookmark;
+import org.openmrs.module.chartsearch.cache.ChartSearchNote;
+import org.openmrs.module.webservices.rest.web.RequestContext;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.annotation.Resource;
+import org.openmrs.module.webservices.rest.web.representation.DefaultRepresentation;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingCrudResource;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+
+/**
+ * REST resource representing a {@link ChartSearchNote}.
+ */
+@Resource(name = RestConstants.VERSION_1 + "/chartsearchnote", supportedClass = ChartSearchBookmark.class, supportedOpenmrsVersions = {"1.10.x", "1.11.*", "1.12.*", "2.0.*"})
+
+public class ChartSearchNoteResource extends DelegatingCrudResource<ChartSearchNote> {
+    @Override
+    public ChartSearchNote getByUniqueId(String uuid) {
+        return Context.getService(ChartSearchService.class).getSearchNoteByUuid(uuid);
+    }
+
+    @Override
+    protected void delete(ChartSearchNote chartSearchNote, String s, RequestContext requestContext) throws ResponseException {
+
```

Comment:
@wluyima I was being consistent with the other resource classes, if that change needs to be made it needs to be across all of them